### PR TITLE
support CONFIG GET parameter [parameter ...]

### PR DESCRIPTION
**Pull request by zeekling**
support CONFIG GET parameter [parameter ...]

### CODE REVIEW

**Review by sazzad16 — CHANGES_REQUESTED** [review]
Add the methods in Pipeline too.

**Comment by zeekling** [comment]
> Add the methods in Pipeline too.

done!

File: src/main/java/redis/clients/jedis/commands/ConfigBinaryCommands.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package redis.clients.jedis.commands;
+
+import java.util.List;
+
+public interface ConfigBinaryCommands {
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/commands/ConfigBinaryCommands.java:5]: These are very few in numbers. We can avoid creating a new interface. Put these in `ConfigCommands`.

**Comment by sazzad16** [comment]
> > Add the methods in Pipeline too.
> 
> done!

I am absolutely sorry. I missed that config commands are not in pipeline. Please remove those.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

**Comment by zeekling** [comment]
> I am absolutely sorry. I missed that config commands are not in pipeline. Please remove those.

removed

File: src/main/java/redis/clients/jedis/commands/ConfigBinaryCommands.java (hunk 1 above)
Comment (zeekling) [reply src/main/java/redis/clients/jedis/commands/ConfigBinaryCommands.java]: > These are very few in numbers. We can avoid creating a new interface. Put these in `ConfigCommands`.

reverted!

File: src/main/java/redis/clients/jedis/TransactionBase.java [hunk 2]
```diff
@@ -3541,8 +3541,8 @@ public String discard() {
   public Response<String> tsDeleteRule(String sourceKey, String destKey) {
     return executeCommand(commandObjects.tsDeleteRule(sourceKey, destKey));
   }
+  // RedisTimeSeries commands
 
-  @Override
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/TransactionBase.java:3545]: `@Override`

File: src/main/java/redis/clients/jedis/TransactionBase.java [hunk 3]
```diff
@@ -3541,8 +3541,8 @@ public String discard() {
   public Response<String> tsDeleteRule(String sourceKey, String destKey) {
     return executeCommand(commandObjects.tsDeleteRule(sourceKey, destKey));
   }
+  // RedisTimeSeries commands
```
Comment (sazzad16) [inline src/main/java/redis/clients/jedis/TransactionBase.java:3544]:
```suggestion
```

File: src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java [hunk 4]
```diff
@@ -184,6 +184,13 @@ public void onCommand(String command) {
   public void configGet() {
     List<String> info = jedis.configGet("m*");
     assertNotNull(info);
+    info = jedis.configGet("*max-*-entries*", "maxmemory");
+    assertNotNull(info);
+    assertEquals(12, info.size());
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java:189]: This is risky. Any future change in Redis may cause this test to fail.

Just test:
- There is at least one entry matching `*max-*-entries*` at even (0-based) position.
- There is `maxmemory` at even (0-based) position.

Or, simply:
- There is `maxmemory` at even (0-based) position.
- Size of the list is at least 4.

File: src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java [hunk 5]
```diff
@@ -184,6 +184,13 @@ public void onCommand(String command) {
...
+    List<byte[]> infoBinary = jedis.configGet("m*".getBytes());
+    assertNotNull(infoBinary);
+    infoBinary = jedis.configGet("*max-*-entries*".getBytes(), "maxmemory".getBytes());
+    assertEquals(12, infoBinary.size());
```
Comment (sazzad16) [inline src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java:193]: Same as `info(String...)`.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/main/java/redis/clients/jedis/TransactionBase.java (hunk 3 above)
Comment (zeekling) [reply src/main/java/redis/clients/jedis/TransactionBase.java]: done

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**